bolt/test/binary-analysis/AArch64/gs-pauth-calls.s

```diff
@@ -429,6 +429,261 @@ bad_indirect_call_mem_chain_of_auts_multi_bb:
         ret
         .size bad_indirect_call_mem_chain_of_auts_multi_bb, .-bad_indirect_call_mem_chain_of_auts_multi_bb
 
+// Tests for CFG-unaware analysis.
+
+        .globl  good_direct_call_nocfg
+        .type   good_direct_call_nocfg,@function
+good_direct_call_nocfg:
+// CHECK-NOT: good_direct_call_nocfg
+        paciasp
+        stp     x29, x30, [sp, #-16]!
+        mov     x29, sp
+
+        bl      callee
+
+        adr     x2, 1f
+        br      x2
+1:
```

Comment:
Maybe it'd be useful to add a comment here that you're using this instruction pattern to make sure BOLT cannot reconstruct the CFG, and this pattern will be used in all test cases for the noCFG analysis?

```
        adr     x2, 1f
        br      x2
1:
```